app/assets/v2/css/search.css

```diff
@@ -0,0 +1,170 @@
+:root {
+  --color-search: 37, 232, 153;
+  --badge-bg: #d6fbeb;
+  --badge-text: #00a55e;
+  --link-color: #3e00ff;
+  --badge-blue-bg: #e7f0fa;
+  --badge-blue-text: #6587ae;
+}
+
+.navbar-nav .nav-link.search__icon {
+  position: relative;
+  font-size: 18px;
+  color: white;
+}
+
+.nav-line {
```

Comment:
Don't restyle bootstrap general classes (or gitcoin general classes) unless you want to change the visual in all the site.